config/i18n-tasks.yml

```diff
@@ -97,6 +97,7 @@ ignore_unused:
   - 'experiments.*'
   - 'event_types.*'
   - 'zxcvbn.*'
+  - 'links.cancel'
```

Comment:
I don't think this is necessary anymore since this entry was removed in this PR, right? I removed this line, then ran `i18n-tasks unused` and `i18n-tasks missing`, and both passed without errors.